src/actions/addTokensToVault.ts

```diff
@@ -0,0 +1,95 @@
+import BN from 'bn.js';
+import { AccountLayout } from '@solana/spl-token';
+import {
+  Vault,
+  SafetyDepositBox,
+  AddTokenToInactiveVault,
+} from '@metaplex-foundation/mpl-token-vault';
+import { Connection, TransactionSignature, PublicKey, Keypair } from '@solana/web3.js';
+
+import { Wallet } from '../wallet';
+import { createApproveTxs } from './shared';
+import { sendTransaction } from './transactions';
+import { CreateTokenAccount } from '../programs';
+import { TransactionsBatch } from '../utils/transactions-batch';
+
+interface Token2Add {
+  tokenAccount: PublicKey;
+  tokenMint: PublicKey;
+  amount: BN;
+}
+
+interface AddTokensToVaultParams {
+  connection: Connection;
+  wallet: Wallet;
+  vaultPub: PublicKey;
```

Comment:
if we use typescript we can omit type hints in the field name

Reply:
upd